test/config/type/function-with-env/function-with-env.test.js

```diff
@@ -9,19 +9,21 @@ describe('function configuration', () => {
         expect(stderr).toBeFalsy();
         expect(stdout).toBeTruthy();
         // Should generate the appropriate files
-        stat(resolve(__dirname, './bin/prod.js'), (err, stats) => {
-            expect(err).toBe(null);
-            expect(stats.isFile()).toBe(true);
-        });
+        expect(existsSync(resolve(__dirname, './bin/prod.js'))).toBeTruthy();
     });
     it('is able to understand a configuration file as a function', () => {
         const { stderr, stdout } = run(__dirname, ['--env', 'isDev']);
         expect(stderr).toBeFalsy();
         expect(stdout).toBeTruthy();
         // Should generate the appropriate files
-        stat(resolve(__dirname, './bin/dev.js'), (err, stats) => {
-            expect(err).toBe(null);
-            expect(stats.isFile()).toBe(true);
-        });
+        expect(existsSync(resolve(__dirname, './bin/dev.js'))).toBeTruthy();
+    });
+    it('is able to understand multiple env flags', () => {
+        const { stderr, stdout } = run(__dirname, ['--env', 'isDev', '--env', 'verboseStats']);
+        expect(stderr).toBeFalsy();
+        expect(stdout).toBeTruthy();
+        expect(stdout).toContain('LOG from webpack.buildChunkGraph.visitModules');
+        // Should generate the appropriate files
+        expect(existsSync(resolve(__dirname, './bin/dev.js'))).toBeTruthy();
```

Comment:
Can we improve testing them here?

Reply:
What can we do? we're already checking for both the env flags

Reply:
Can you give example?

Reply:
I think for better testing will be great to use `DefinePlugin`, insert this variables to source code and test what we contains them in generated bundle

Reply:
Thanks for the suggestion @evilebottnawi 👍 Done 💯